packages/editor/src/components/browser/CmsBrowser.tsx

```diff
@@ -1,15 +1,151 @@
-import { useState } from 'react';
-import { Input } from '../widgets';
+import { useMemo, useState, useEffect } from 'react';
+import { Collapsible, ExpandableCell, MessageRow, MessageText, SelectRow, Table, TableCell } from '../widgets';
 import { UseBrowserImplReturnValue } from './useBrowser';
+import { useEditorContext, useMeta } from '../../context';
+import {
+  ColumnDef,
+  ExpandedState,
+  RowSelectionState,
+  flexRender,
+  getCoreRowModel,
+  getExpandedRowModel,
+  getFilteredRowModel,
+  useReactTable
+} from '@tanstack/react-table';
+import { ContentObject } from '@axonivy/inscription-protocol';
 
 export const CMS_BROWSER_ID = 'cms' as const;
 
 export const useCmsBrowser = (): UseBrowserImplReturnValue => {
-  const [value, setValue] = useState('cms');
+  const [value, setValue] = useState('');
 
-  return { id: CMS_BROWSER_ID, name: 'CMS', content: <CmsBrowser value={value} onChange={setValue} />, accept: () => value };
+  return {
+    id: CMS_BROWSER_ID,
+    name: 'CMS',
+    content: <CmsBrowser value={value} onChange={setValue} />,
+    accept: () => createFullPathCMS(value)
+  };
 };
 
-const CmsBrowser = (props: { value: string; onChange: (value: string) => void }) => {
-  return <Input {...props} />;
+export const createFullPathCMS = (value: string) => {
```

Comment:
this method does not calculate the full path, but add the ivy cms co. So rename it to what it does.